sh/tests/auto-exec-bin_test.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+FILENAME="student/auto-exec-bin.sh"
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+setupbin() {
```

Comment:
this is checking if auto-exec-bin.sh exists
If it exist the myBins folder will not be created right?

Reply:
yes

Reply:
yes i have a bug here, I have a uncorrected condition
fixed by checking for the binary